app/javascript/components/community/video-grid/useVideoGrid.tsx

```diff
@@ -55,9 +55,9 @@ export function useVideoGrid(
     usePaginatedRequestQuery<APIResponse>(
       [
         'community-video-grid-key',
-        request.query.criteria,
-        request.query.videoTrackSlug,
-        request.query.videoPage,
+        request.query.criteria || '',
+        request.query.videoTrackSlug || '',
+        request.query.videoPage || 1,
```

Comment:
What I'd like to understand is what's changed in this PR to cause this to hurt us?

Reply:
It might me the pagination change

Reply:
I'll double check. I tried to revert a change there.

Reply:
Found it: https://github.com/exercism/website/pull/5907/commits/778c9e86bc192b41b0bb1032cf7ac46912c0f02f